aquacomputer_d5next.c

```diff
@@ -654,6 +691,413 @@ static void aqc_debugfs_init(struct aqc_data *priv)
 
 #endif
 
+
+static int aqc_fan_curve_show(struct device *dev, struct device_attribute *attr, int offset)
+{
+	struct sensor_device_attribute_2 *sensor_attr = to_sensor_dev_attr_2(attr);
+	struct aqc_data *priv = dev_get_drvdata(dev);
+	int channel = sensor_attr->nr;
+	int idx = sensor_attr->index;
+	u16 to_get;
+
+	if (channel > priv->num_fans)
```

Comment:
same problem with channel number checking

Reply:
fixed in 0be96963, thanks!